plugins/search/src/components/SearchModal/SearchModal.tsx

```diff
@@ -41,6 +41,10 @@ import { useNavigateToQuery } from '../util';
 export interface SearchModalProps {
   open?: boolean;
   toggleModal: () => void;
+  searchBarProps?: Omit<
+    SearchBarBaseProps,
+    'value' | 'className' | 'onChange' | 'onSubmit'
```

Comment:
Is there a reason to omit the `className`?

Reply:
There's already `classes.input` passed as className for SearchBar. Do you think `classnames` should be used and allow to extend className ?

Reply:
Could be that this should be unified - i.e. that people should move over to passing in `SearchBarBaseProps.className` instead of `classes.input`? But maybe in a separate PR if so, what you do here is probably OK for the current state of the code around your change